File: solutions/0001-two-sum/solution.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> index = {0, 0};
        for (auto i = nums.begin(); i < nums.end(); i++) {
            for (auto j = i + 1; j < nums.end(); j++) {
                if (*i + *j == target) {
                    index[0] = (i - nums.begin());
                    index[1] = (j - nums.begin());
                    return index;
                }
            }
        }
    return index;
    }
};

File: solutions/0001-two-sum/solution.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n = nums.length;
        HashMap <Integer,Integer> map = new HashMap<>();
        for(int i =0;i<n;i++){
            int rem = target - nums[i];
            Integer res = map.get(rem);
            if(res != null && res != i){
                return new int[]{i,res};
            }
            map.put(nums[i],i);
        }
        return null;
    }
}
